### ISSUE TITLE
Question on wishlist item jest

### ISSUE THREAD

**Issue by Mark1626** [denolib/wishlist#7]
I was exploring the community and came upon this request, thought I can help out.

I don't think completely porting `jest` to `deno` is a solution. Maybe a jest `test-runner` for deno? Would be similar to this https://github.com/MaximeHeckel/jest-runner-go

Even so, looks like `deno-std` has a testing module, so I think helping out there can be better

https://github.com/denoland/deno_std/tree/master/testing
https://github.com/denoland/deno_std/pull/516

Can someone mention if this wish-list item is still needed, if so let me know I'll see if I can help

**Comment by kevinkassimo** [comment]
I believe current focus should be effort to improve https://github.com/denoland/deno_std/tree/master/testing

**Comment by ahuggins-nhs** [comment]
I agree that focus should be on tooling in Deno core, but I also believe it should be as easy as possible for developers to migrate to Deno. I'm not personally going to work on Jest, but I am working on Mocha and have written a working test runner: https://github.com/ahuggins-nhs/deno_mocha.

**Comment by binyamin** [comment]
Bumping this issue. It's 2022, and Deno's testing module has evolved a bunch.
